.github/workflows/image-builder.yml

```diff
@@ -1,62 +0,0 @@
-# Build a docker image on push to main
```

Comment:
How do we build the image to use for this version?

Reply:
we don't, Yang will handle it on his side with his fancy new Jenkins setup